tests/lightning_tensor/test_expval.py

```diff
@@ -0,0 +1,398 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the expectation value calculations on the LightningTensor device.
```

Comment:
This appears to be mostly a duplicate of `tests/test_expval.py`. Could we install `lightning.tensor`, import the device and just run `tests/test_expval.py`; skipping tests or xfailing where it makes sense?

Reply:
I totally agree. As for the previous PR, tests should be updated separately (as the entry point in the `setup.py` file). We have two separate stories for that. So far, I have mostly copied the tests in that folder to make sure that the device works correctly, but the entire architecture of tests will be updated ASAP